.travis.yml

```diff
@@ -61,6 +61,16 @@ matrix:
       services: docker
       before_install:
       addons:
+    - sudo: required
+      env: TOXENV=le_auto_centos6
+      services: docker
+      before_install:
+      addons:
+    - sudo: required
```

Comment:
Because of my comment [here](https://github.com/certbot/certbot/pull/2635/files#r87292919), tests on CentOS 7 should be removed.